esp/esp/program/modules/handlers/schedulingcheckmodule.py

```diff
@@ -47,7 +47,7 @@ def format_table(self, l, options={}, help_text=""):
     def format_list(self, l, options={}, help_text=""):
         return l
 
-class HTMLSCFormatter:
+class HTMLSCFormatter: # This might be dead, but idk maybe someone still wants it
```

Comment:
I think it's specific enough to this context, and simple enough, that you can just nuke it.  If anyone else wants it they can just grab it from git.